lib/devise/encryptable/encryptors/pbkdf2.rb

```diff
@@ -0,0 +1,14 @@
+module Devise
+  module Encryptable
+    module Encryptors
+      class Pbkdf2 < Base
+        def self.digest(password, stretches, salt, pepper)
+          digest = OpenSSL::Digest::SHA512.new
+          len = digest.digest_length
+          iterations = 2**stretches
```

Comment:
Where does stretches come from? If this is a value that's usually used for bcrypt's work factor (for example), this formula seems strange to me. a bcrypt work factor of 12 is standard, but that would result in only a 4096 iterations, which is insufficient.